app/(afterLogin)/(employeeInformation)/employees/manage-employees/page.tsx

```
@@ -21,6 +30,14 @@
     setOpen(false);
   };
 
+  const handleDownloadUserData = (downloadFormat: string) => {
+    // Convert searchParams to Record<string, string>
+    const params: Record<string, string> = Object.fromEntries(
+      Object.entries(searchParams).map(([k, v]) => [k, v == null ? '' : String(v)])
```

ESLint

[error] Replace `k,·v·==·null·?·''·:·String(v)])` with `⏎········k,⏎········v·==·null·?·''·:·String(v),⏎······]),`

(prettier/prettier)